core/src/main/java/com/yahoo/imapnio/async/internal/CompressCommand.java

```diff
@@ -13,12 +14,16 @@
     private static final String COMPRESS_DEFLATE = "COMPRESS DEFLATE";
 
     /**
-     * Initializes the @{code COMPRESS_DEFLATECommand}. Constructor is only visible to this package so it can only be called by @{code
-     * ImapAsyncSessionImpl}.
+     * Initializes the @{code CompressCommand}. Constructor is only visible to this package so it can only be called by @{code ImapAsyncSessionImpl}.
      * 
      * @see "RFC 4978"
      */
     CompressCommand() {
         super(COMPRESS_DEFLATE);
```

Comment:
What is the difference between this and IMAPCommandType?

Reply:
that is the command string, for example, "COMPRESS DEFLATE", that is used for the command sent to server.  It is not an enum.